test/conformance/memory/urMemImageCreate.cpp

```diff
@@ -83,56 +91,68 @@ TEST_P(urMemImageCreateTest, InvalidSize) {
                                       nullptr, &image_handle));
 }
 
-TEST_P(urMemImageCreateTest, InvalidImageDesc) {
```

Comment:
I split up this test into individual smaller ones - I found while diagnosing the CUDA adapter finding which specific assertion was failing was unhelpful.